### DISCUSSION TITLE
Running on Prod node before Duncan?

### DISCUSSION THREAD

**Discussion by Moataz-E** [paradigmxyz/reth#6782] [General]
Hi,

I will be migrating my validators to a new machine away from geth. The plan is to run erigon as the execution client but I would much prefer a rust-based client like reth as a long term solution.

Any chance a beta client will be out before the Duncan upgrade goes live?

Thanks.

**Comment by mattsse** [comment]
beta is planned for next week, it will include a few breaking db changes, so not recommending starting a new sync rn.

although current release already has full cancun support

**Reply by Moataz-E** [reply]
Thanks, I will wait for beta. Looking forward to it!
